packages/compose/README.md

```diff
@@ -326,7 +326,7 @@ Dispatches a bubbling focus event when the iframe receives focus. Use
 
 _Returns_
 
--   `Object`: Ref to pass to the iframe.
+-   Ref to pass to the iframe.
```

Comment:
Nit: would it make more sense to swap `Object` with `RefCallback`, instead of just removing it?

Reply:
Updated. Since the docs are autogenerated I added a return type in 266fdc5b217a31ce7358058889b2086faf3f13e8

Reply:
Cool, thank you!